run-bench.sh

```diff
@@ -1,74 +1,103 @@
 usage() {
-    echo
-    echo "nsuite benchmark runner:"
-    echo
-    echo "run with one, and only one, of the following options:"
-    echo "   arbor  : run arbor benchmarks"
-    echo "   neuron : run neuron benchmarks"
-    echo "   coreneuron : run coreneuron benchmarks"
-    echo
+    cat <<_end_
+Usage: run-bench.sh [--prefix=PATH] [--model MODEL] [--config CONFIG] SIMULATOR
```

Comment:
Again, the formatting of the new argument is inconsistent with the existing arguments.

Let's just pick one approach and stick with it (I prefer without the `=`).

Reply:
Fixed (currently with '=', but still happy to go the other way).